docs/advanced-usage/batch-logs.md

```diff
@@ -3,27 +3,31 @@ title: Batch Logs
 weight: 3
 ---
 
-In some cases you may want to link multiple activities back to a single activity that started it all. For example author was deleted, that cascades soft deletes to the books for that author.
-Now the deleted author and deleted books are now having the same `uuid`.
+In some situations you may want to process multiple activities back to a single batch activity.
 
-You can start new batch by calling `LogBatch::startBatch()` before any activity is done then all of following activities will link back by UUID to that batch. After finishing activities you should end the batch `LogBatch::endBatch()`.
+For example when a `User` deletes an `Author`, then that cascades soft deletes to the `Book`s that were owned by the `Author`. This way all modifications caused by that initial action are still associated with the same causer and batch UUID.
+
+To start a new batch call `LogBatch::startBatch()` before any activity is done. Then all following actions will link back by UUID to that batch. After finishing activities you should end the batch by calling `LogBatch::endBatch()`.
+
+Here's an example:
 
 ```php
 use Spatie\Activitylog\Facades\LogBatch;
 use Spatie\Activitylog\Models\Activity;
 
 LogBatch::startBatch();
-$article = NewsItem::create(['name' => 'new article']);
-$article->update(['name' => 'update article']);
-$article->delete();
+$author = Author::create(['name' => 'Philip K. Dick']);
+$book = Book::create(['name' => 'A Scanner Brightly', 'author_id' => $author->id]);
+$book->update(['name' => 'A Scanner Darkly']);
+$author->delete();
 
-$batchUuid = LogBatch::getUuid(); // save batch id to retrive activities later
+$batchUuid = LogBatch::getUuid(); // save batch id to retrieve activities later
 LogBatch::endBatch();
 
-Activity::forBatch($batchUuid)->get(); // collection of 3 activity models ['created', 'updated', 'deleted']
+Activity::forBatch($batchUuid)->get(); // TODO: unsure what the output would be...
```

Comment:
// collection of 3 activity models ['created', 'updated', 'deleted']

Reply:
Sorry - I updated the example to match the `Author` and `Book` example used in the issue on the spatie repo. Also this maybe outdated for what I sent over.


I planned to setup a quick laravel app with Author and Book models to run that code.

Reply:
No need for full output here, alternatively I described it here.

Reply:
Ahh - I assumed that as I modified the example the activity changed too.

1. Author created
2. Book created
3. Book updated
4. Author deleted
5. Book deleted - via cascade

Reply:
cool, I think describing it like this sounds much better.

Reply:
Hmmm - so I updated the note to include the info in a similar format to this...unfortunately it's not working as expected.
In total the example code -from my limited understanding- should have included 7 activities total. The obvious 5 obvious ones shown in the code itself. And then 2 deletes for the books that occurs as a cascade delete via a delete hook.

```php
    $author = Author::create(['name' => 'Philip K. Dick']); // 1

    $book = Book::create(['name' => 'A Scanner Brightly', 'author_id' => $author->id]); // 2
    $book->update(['name' => 'A Scanner Darkly']); // 3

    $book2 = Book::create(['name' => 'Paycheck', 'author_id' => $author->id]); // 4


    $author->delete(); // 5 (and 6, 7 via the cascade deleted books)
```

The delete hook is:
```php
    public static function boot() {
        parent::boot();

        static::deleting(function($author) {
            $author->books()->delete();
        });
    }
```

Maybe I misunderstood how the feature was implemented compared to the impression I got from the docs and the description in the issue? Or maybe I'm just doing something wrong, in which case we can add more to the docs to prevent that issue. 😄

Reply:
Have you Added `LogsActivity` trait on the author model? 
I tickled a little with the tests it turns out that we have to add that trait to mark the model as loggable.